cmds/minigames/connect4.js

```diff
@@ -175,288 +289,320 @@ class connect4 extends two_p_Game{
+            stack.forEach((col) => {
+                if (stack[col] > -1) {
+                    board[stack[col]][col] = "🟡";
                     stack[col] -= 1;
-                    let {score,move} = this.c4minimax(board,stack,depth-1,false,col,stack[col]+1)
-                    if(score > best_score){
-
-                        best_score = score
-                        best_move = col
-
+                    let { score, move } = this.c4minimax(
+                        board,
+                        stack,
+                        depth - 1,
+                        false,
+                        col,
+                        stack[col] + 1
+                    );
+                    if (score > best_score) {
+                        best_score = score;
+                        best_move = col;
                     }
                     stack[col] += 1;
-                    board[stack[col]][col] = "⚪"
-
+                    board[stack[col]][col] = "⚪";
                 }
-                
             });
 
-            return {score : best_score, move : best_move}
-
-        }else{
+            return { score: best_score, move: best_move };
+        } else {
+            let cbest_score = Infinity;
+            let cbest_move = 0;
 
-            let cbest_score = Infinity
-            let cbest_move = 0
-
-            stack.forEach( col => {
-
-                if(stack[col] > -1){
-
-                    board[stack[col]][col] = "🔴"
+            stack.forEach((col) => {
+                if (stack[col] > -1) {
+                    board[stack[col]][col] = "🔴";
                     stack[col] -= 1;
-                    let {score, move} = this.c4minimax(board,stack,depth-1,true,col,stack[col]+1)
-                    if(score < cbest_score){
-
-                        cbest_score = score
-                        cbest_move = col
-
+                    let { score, move } = this.c4minimax(
+                        board,
+                        stack,
+                        depth - 1,
+                        true,
+                        col,
+                        stack[col] + 1
+                    );
+                    if (score < cbest_score) {
+                        cbest_score = score;
+                        cbest_move = col;
                     }
                     stack[col] += 1;
-                    board[stack[col]][col] = "⚪"
-
+                    board[stack[col]][col] = "⚪";
                 }
-                
             });
 
-            return {score : cbest_score, move : cbest_move}
-
+            return { score: cbest_score, move: cbest_move };
         }
-
-
-
     }
 
-    board_evaluation(board, color){
-
-        let result = {two : 0, three : 0, four : 0}
-
-        for(var i = 0; i < 6; i++){
-            for(var j = 0; j < 7; j++){
-
-                if(board[i][j] === color){
+    // not sure what this does, but simplify it if possible
+    board_evaluation(board, color) {
```

Comment:
Not sure what this does, but there is a large chunk of code block that looks identical, and complex nested for loop, if and switch statement
Simplify it if possible